sources/Application/Views/ModalDialogs/SelectProjectDialog.cpp

```diff
@@ -282,32 +281,27 @@ void SelectProjectDialog::setCurrentFolder(Path &path) {
 
 	I_Dir *dir=FileSystem::GetInstance()->Open(currentPath_.GetPath().c_str()) ;
 
-  if (dir) 
-  {
-
+	if (dir) 
+	{
 		// Get all lgpt something
-
 		dir->GetContent("*");
-    dir->Sort();
-      
+		// dir->Sort();
```

Comment:
Should we keep this and define an empty sort method for the moment? (possibly with a log stating sorting is not implemented yet)

I'd leave this file untouched, most changes are unrelated

Reply:
good point! will do.

Reply:
actually this is still using the older I_Dir so it can still do the sorting